CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 Changelog
 ================================================================================
 
+v2406.0-1
```

Comment:
The last entry referenced a "release version" (2404.1). This entry references only a "major version" (2406.0), so I guess there is a ".0" missing here.

Reply:
Yes, seems, we need to extend the version number. Based on the chosen versions, it should be 2406.0.1-1, though.

Adjusted and also extended the changelog a little.